tests/test_common_options.py

```diff
@@ -15,6 +15,17 @@ def test_list_targets():
         assert i in output
 
 
+# Polarion TC MTA-472
+def test_list_targets_duplicates_bug_mta_2064():
```

Comment:
Let's put the bug number in a comment rather than in TC name .

def test_list_targets_duplicates():
# Polarion TC MTA-472, Automates Bug MTA 2064